ui/raidboss/data/00-misc/test.js

```diff
@@ -196,10 +196,10 @@
       regexKo: Regexes.echo({ line: 'cactbot 언어.*?', capture: false }),
       infoText: function(data) {
         return {
-          en: 'Language: ' + data.lang,
-          de: 'Sprache: ' + data.lang,
-          cn: '语言: ' + data.lang,
-          ko: '언어: ' + data.lang,
+          en: 'Language: ' + data.parserLang,
+          de: 'Sprache: ' + data.parserLang,
```

Comment:
I do not know if it is useful, but if the translation is missing, here is the line.
`fr: 'Langage: ' + data.parserLang,`